CMakeLists.txt

```diff
@@ -1,8 +1,7 @@
-cmake_minimum_required(VERSION 3.5)
+cmake_minimum_required(VERSION 3.10.2)
```

Comment:
Why not just `3.10`?

Reply:
I have not tested `3.10.1`, it is possible that there is something wrong. Also, I read somewhere, that cmake adds features to the patches, not just bug-fixes.